interface/forms/group_attendance/functions.php

```diff
@@ -109,7 +109,7 @@ function insert_patient_appt($pid, $gid, $pc_aid, $pc_eventDate, $pc_startTime,
         $insert_sql =
             "INSERT INTO openemr_postcalendar_events " .
             "(pc_catid, pc_aid, pc_pid, pc_gid, pc_title, pc_informant, pc_eventDate, pc_recurrspec, pc_startTime, pc_sharing, pc_apptstatus) ".
-            "VALUES (1000, ?, ?, ?, 'Group Therapy', 1, ?, ?, ?, 0, ?); ";
+            "VALUES (15, ?, ?, ?, 'Group Therapy', 1, ?, ?, ?, 0, ?); ";
```

Comment:
should not hard-code 15 (for now would just insert the pc_catid of the lowest id you get when query 'SELECT pc_catid FROM openemr_postcalendar_categories WHERE pc_cattype = 3 AND pc_active = 1' )